src/main/java/nl/novi/techiteasy1121/services/CIModuleService.java

```diff
@@ -0,0 +1,86 @@
+package nl.novi.techiteasy1121.services;
+
+import nl.novi.techiteasy1121.dtos.CIModuleDto;
+import nl.novi.techiteasy1121.exceptions.RecordNotFoundException;
+import nl.novi.techiteasy1121.models.CIModule;
+import nl.novi.techiteasy1121.repositories.CIModuleRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+// Deze klasse bevat de service methodes van de CIModuleController
+
+@Service
+public class CIModuleService  {
+
+
+    private CIModuleRepository ciModuleRepository;
+
+    public CIModuleService(CIModuleRepository ciModuleRepository) {
+        this.ciModuleRepository = ciModuleRepository;
+    }
+
+    public List<CIModuleDto> getAllCIModules() {
+        List<CIModule> ciModules = ciModuleRepository.findAll();
+        List<CIModuleDto> dtos = new ArrayList<>();
+        for (CIModule ci : ciModules) {
+            dtos.add(transferToDto(ci));
+        }
+        return dtos;
+    }
+
+    public CIModuleDto getCIModule(long id) {
```

Comment:
De service-klasse voor CIModule is goed gestructureerd en de methoden zijn eenvoudig te begrijpen. Het gebruik van DTO's om gegevens tussen de lagen te transporteren is een goede keuze.